src/winrt/activation/catalog.cpp

```diff
@@ -4,6 +4,10 @@
 #include <roapi.h>
 #include <winstring.h>
 #include <rometadataresolution.h>
+#include <combaseapi.h>
+#include <wrl.h>
+#include <msxml6.h>
```

Comment:
Ideally you should use XmlLite rather than msxml.

Reply:
Isn't XmlLite asynchronous?  Async will cause issues since we need to have this completed early.

Reply:
No, it is not. https://docs.microsoft.com/en-us/archive/msdn-magazine/2007/april/xmllite-a-small-and-fast-xml-parser-for-native-c

Reply:
sorry, dinosaur here. I steered them towards MSXML, but that would be a much better alternative.

Reply:
Switched over to XmlLite